phpunit.xml.dist

```diff
@@ -37,7 +37,7 @@
         <server name="PICKPOINT_API_KEY" value="YOUR_API_KEY" />
         <server name="TOMTOM_MAP_KEY" value="YOUR_MAP_KEY" />
         <server name="GEOCODE_EARTH_API_KEY" value="YOUR_GEOCODE_EARTH_API_KEY" />
-        <server name="IPFINDER_API_KEY" value="YOUR_API_KEY" />
+        <server name="IPFINDER_API_KEY" value="ent" />
```

Comment:
Why this has been changed ?
It should be `YOUR_API_KEY`.

Reply:
Thanks I forgot